pkgs/development/libraries/libvirt/default.nix

```diff
@@ -114,13 +114,13 @@ stdenv.mkDerivation rec {
   # NOTE: You must also bump:
   # <nixpkgs/pkgs/development/python-modules/libvirt/default.nix>
   # SysVirt in <nixpkgs/pkgs/top-level/perl-packages.nix>
```

Comment:
Is the comment about SysVirt in perl-packages.nix outdated? If so (SysVirt shouldn't be updated), please update/remove the comment in commit 1, then have the version bump as commit 2.

Reply:
SysVirt doesn't have a release since 9.7.0, we often don't bump it because the devs just don't create a release.

Reply:
I think the comment is good to have, just the use of the word "must" is wrong.

Reply:
Ok.